packages/atomic/src/components/commerce/atomic-product-children/atomic-product-children.ts

```diff
@@ -10,28 +10,35 @@ import {withTailwindStyles} from '@/src/decorators/with-tailwind-styles.js';
 import {filterProtocol} from '../../../utils/xss-utils';
 import {renderButton} from '../../common/button';
 import type {CommerceBindings} from '../atomic-commerce-interface/atomic-commerce-interface';
-import styles from './atomic-product-children.tw.css';
 import type {SelectChildProductEventArgs} from './select-child-product-event';
 import '../atomic-commerce-text/atomic-commerce-text';
-import {injectStylesForNoShadowDOM} from '@/src/decorators/inject-styles-for-no-shadow-dom';
+import {bindingGuard} from '@/src/decorators/binding-guard';
 import {multiClassMap, tw} from '@/src/directives/multi-class-map';
 
+/**
+ * Maximum number of child products to display before showing a "+N" button.
+ * This limit ensures a consistent UI and prevents layout issues with large product variants.
+ */
+const AMOUNT_OF_VISIBLE_CHILDREN = 5;
+
 /**
  * The `atomic-product-children` component renders a section that allows the user to select a nested product (e.g., a color variant of a given product).
  *
  * This component leverages the [product grouping](https://docs.coveo.com/en/l78i2152/) feature.
- * @alpha
+ *
+ * By default, the component displays up to 5 child products before showing a "+N" button for additional variants.
```

Comment:
```suggestion
 * The component displays up to 5 child products before showing a "+N" button for additional variants.
```

"By default" makes it seem like that behavior can be customized which isn't really the case.